README.md

```diff
@@ -90,19 +90,19 @@ _You can find the version on the [releases](https://github.com/ably/ably-asset-t
 - `ABLY_API_KEY`: Used by all example apps to authenticate with Ably using basic authentication. Not recommended in production, and can be taken from [here](https://ably.com/accounts).
 - `MAPBOX_ACCESS_TOKEN`: Used to access Mapbox Navigation SDK/ APIs, and can be taken from [here](https://account.mapbox.com/). Using the Mapbox token is only required to run the **Publisher** example apps.
 - Open `AblyAssetTracking.xcworkspace` to open an Xcode workspace containing the Subscriber example app and the Swift Package containing the SDKs that showcase how to use the subscriber part of the Ably Asset Tracking SDKs.
-- Open `PublisherExampleSwiftUI.xcodeproj` to open an Xcode workspace containing the Publisher exmple app
+- Open `PublisherExample.xcodeproj` to open an Xcode workspace containing the Publisher exmple app
```

Comment:
Shall we please also update these instructions with our recent findings about how to run the two apps?

Reply:
I've created https://github.com/ably/ably-asset-tracking-swift/pull/652 for this.